grid-proxy/tools/db/generate.go

```diff
@@ -347,6 +347,7 @@ func generateNodes(db *sql.DB) error {
 				State:  powerState[rand.Intn(len(powerState))],
 				Target: powerState[rand.Intn(len(powerState))],
 			},
+			has_gpu: i%2 == 0,
```

Comment:
feel free to ignore this, but wouldn't it be better to kind of randomize the number of nodes that have gpus?